src/tracer/src/common/target_process/yml_rules/tracer.exclude.yml

```diff
@@ -0,0 +1,18 @@
+rules:
+  - rule_name: "Exclude version"
+    display_name: "Exclude version"
+    condition:
+      or:
+        - command_contains: "--version"
+        - command_contains: "-version"
+        - command_contains: "-v"
+        - command_contains: "--v"
+
+  - rule_name: "Exclude help"
+    display_name: "Exclude help"
+    condition:
+      or:
+        - command_contains: "--help"
+        - command_contains: "-help"
+        - command_contains: "-h"
```

Comment:
I would not include -h or --h for same reason as above - some programs use -h for other options. For example, in `samtools view -h`, the -h means to include the header in the output.